react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -172,8 +172,20 @@ const AutocompleteInput = ({
   })
 
   return (
+    <form action="#" onSubmit={e => {
+      e.preventDefault()
+      e.stopPropagation()
+      e.nativeEvent.stopImmediatePropagation()
+    }}>
     <div className={handles.autoCompleteOuterContainer}>
       <div className={classContainer}>
+      <InputSearch 
+          ref={inputRef}
+          size="large"
+          value={value}
+          {...restProps}          
+        />
+        {/*        
         <Input
```

Comment:
I think we can use `<Input type="search" ...` instead of using InputSearch. Changing to `InputSearch` would break current CSS customisations.

Reply:
Ok I try it ant I'll do it something later.